modules/ROOT/pages/coverage-maven-concept.adoc

```diff
@@ -43,7 +43,7 @@ To run coverage reports in Studio see xref:coverage-studio-concept.adoc[Using Co
                 <format>console</format>
                 <format>html</format>
             </formats>
-        </coverage>
+        </>
```

Comment:
why are we removing "coverage" word here?

Reply:
What a great catch! 
It seems I deleted it by mistake.
Thanks for mentioning it!